x/ccv/democracy/staking/module.go

```diff
@@ -29,20 +29,22 @@ type AppModule struct {
 	// embed the Cosmos SDK's x/staking AppModule
 	staking.AppModule
 
-	keeper     keeper.Keeper
-	accKeeper  types.AccountKeeper
-	bankKeeper types.BankKeeper
+	keeper         keeper.Keeper
+	accKeeper      types.AccountKeeper
+	bankKeeper     types.BankKeeper
+	consumerKeeper ConsumerKeeper
 }
 
 // NewAppModule creates a new AppModule object using the native x/staking module
 // AppModule constructor.
-func NewAppModule(cdc codec.Codec, keeper keeper.Keeper, ak types.AccountKeeper, bk types.BankKeeper) AppModule {
+func NewAppModule(cdc codec.Codec, keeper keeper.Keeper, ak types.AccountKeeper, bk types.BankKeeper, ck ConsumerKeeper) AppModule {
```

Comment:
Same question as above on why consumer keeper is now passed through constructor

Reply:
This is to use preCCV flag of consumer keeper while sovereign is live. I think it's fine to remove this.